docs/reference/filter_field_definition.rst

```diff
@@ -10,29 +10,29 @@ These fields are displayed inside the filter box. They allow you to filter the l
 
 A filter instance is always linked to a Form Type, there are 7 types available:
 
-* `Sonata\AdminBundle\Form\Type\Filter\NumberType`: display 2 widgets, the operator ( >, >=, <=, <, =) and the value,
-* `Sonata\AdminBundle\Form\Type\Filter\ChoiceType`: display 2 widgets, the operator (yes and no) and the value,
-* `Sonata\AdminBundle\Form\Type\Filter\DefaultType`: display 2 widgets, an hidden operator (can be changed on demand) and the value,
-* `Sonata\AdminBundle\Form\Type\Filter\DateType`: display 2 widgets, the operator ( >, >=, <= , <, =) and the value,
-* `Sonata\AdminBundle\Form\Type\Filter\DateRangeType`: display 3 widgets, the operator (between and not between) and the two values,
-* `Sonata\AdminBundle\Form\Type\Filter\DateTimeType`: display 2 widgets, the operator ( >, >=, <= , <, =) and the value,
-* `Sonata\AdminBundle\Form\Type\Filter\DateTimeRangeType`: display 3 widgets, the operator (between and not between) and the two values,
-
-The `Form Type` configuration is provided by the filter itself.
+* ``Sonata\AdminBundle\Form\Type\Filter\NumberType``: display 2 widgets, the operator ( >, >=, <=, <, =) and the value,
+* ``onata\AdminBundle\Form\Type\Filter\ChoiceType``: display 2 widgets, the operator (yes and no) and the value,
```

Comment:
```suggestion
* ``Sonata\AdminBundle\Form\Type\Filter\ChoiceType``: display 2 widgets, the operator (yes and no) and the value,
```

Reply:
The minimum changes for what it's worth in this PR are working. Thank you.